Antidote/AbstractCallViewController.h

```diff
@@ -8,31 +8,45 @@
 
 #import <UIKit/UIKit.h>
 
+#import "OCTToxAVConstants.h"
+
 @class OCTSubmanagerCalls;
 @class OCTChat;
 @class OCTCall;
 
+@protocol AbstractCallViewControllerDelegate <NSObject>
+
+@required
+- (void)dismissCurrentCall;
+- (void)callAccept;
+- (BOOL)sendCallControl:(OCTToxAVCallControl)control error:(NSError **)error;
+
+- (void)otherCallAccept:(BOOL)accept;
+
+@property (nonatomic, assign) BOOL enableMicrophone;
+
+@end
```

Comment:
So, for example, when user pressed `mute` button following happens:
- controllers sends `activeCallMuteButtonPressed` message to CallsManager
- CallsManager sends mute or unmute control to objcTox
- CallsManager sets `muteSelected` property of controller to new value
- controller in setter `- (void)setMuteSelected:(BOOL)selected` updates UI.

Here is the flow for another incoming call:
- we have active call
- CallsManager gets another incoming call
- CallsManager
  - sets `showIncomingCallView` to `YES`
  - sets `incomingCallCallerName` to caller's name
- ...
- user taps answer button
- controllers sends `activeCallAnswerIncomingCallButtonPressed` to CallsManager
- CallsManager changes active call
- CallsManager sets `showIncomingCallView` to `NO`
- CallsManager sets `callerName` to new caller's name and updates `callDuration`
- CallsManager sets `pausedCallsArray` to new array with old call object
- controller in `- (void)setPausedCallsArray:` setter checks if `pausedCallsArray` is empty
- it isn't empty so controller shows top right button with badge (1)
- when clicking on top right button controller shows paused table with old call object.